app/models/sir_trevor_rails/blocks/displayable.rb

```diff
@@ -14,7 +14,7 @@ def item_ids
       end
 
       def ordered_items
-        items.sort_by { |x| x[:weight] }.map { |x| x[:id] }
+        items.sort_by { |x| x[:weight].to_s.rjust(5, '0') }.map { |x| x[:id] }
```

Comment:
```suggestion
        items.sort_by { |x| x[:weight].to_i) }.map { |x| x[:id] }
```

Could we just make sure to parse the weight as an integer?

Reply:
Yes, I like that solution. In fact I thought about a cast to integer as well, but I'm relatively new to ruby and so I was not sure if there might be cases when this can fail or raise an exception.